docs/runbooks/KubeVirtVMHighCPUUsage.md

```diff
@@ -0,0 +1,144 @@
+# KubeVirtVMHighCPUUsage
+
+## Meaning
+
+This alert fires when a Virtual Machine (VM) has been consuming more than 80% of its allocated CPU resources for more than 5 minutes.
+
+## Impact
+
+High CPU usage in a VM can indicate:
+- **Performance degradation** - Applications running inside the VM may become slow or unresponsive
+- **Resource contention** - The VM may be competing for CPU resources with other workloads
```

Comment:
How can you tell this from the CPU load inside the VM?

Reply:
We cant, This needs to be examined from the node POV

Reply:
But how would that trigger high CPU usage inside the VM?